training/backends/kubernetes.js

```diff
@@ -44,14 +44,14 @@ const watcher = new class JobWatcher extends Tp.RefCounted {
     async _doOpen() {
         let currentJobs;
         try {
-            currentJobs = (await k8sApi.listNamespacedJob(Config.TRAINING_KUBERNETES_NAMESPACE.
+            currentJobs = (await k8sApi.listNamespacedJob(Config.TRAINING_KUBERNETES_NAMESPACE,
                 false /* includeUninitialized */,
                 false /* pretty */,
                 undefined /* _continue */,
                 undefined /* fieldSelector */,
                 this._computeLabelSelector())).body;
-        } catch(res) {
-            throw new Error(`Failed to list Kubernetes jobs: ${res.response.body.message}`);
+        } catch(err) {
+            throw new Error('Failed to list Kubernetes jobs:' +  JSON.stringify(err));
```

Comment:
err sometimes does not return a res.  And err.toString() does not output helpful error messages `[object object]` .